_analyzers/token-filters/remove-duplicates.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: default
+title: Remove duplicate
+parent: Token filters
+nav_order: 350
+---
+
+# Remove duplicate token filter
+
+The `remove_duplicates` token filter is used to remove duplicate tokens that are generated at the same position during analysis.
+
+## Example
+
+The following example creates an index with a `keyword_repeat` token filter. The filter adds a `keyword` version of each token at the same position as the token itself and then uses a `kstem` to create a stemmed version of the token:
+
+```json
+PUT /example-index
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "custom_analyzer": {
+          "type": "custom",
+          "tokenizer": "standard",
+          "filter": [
+            "lowercase",
+            "keyword_repeat",
+            "kstem"
+          ]
+        }
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Use the following request to analyze the string `Slower turtle`:
+
+```json
+GET /example-index/_analyze
+{
+  "analyzer": "custom_analyzer",
+  "text": "Slower turtle"
+}
+```
+{% include copy-curl.html %}
+
+The response contains the token `turtle` twice in the same position:
```

Comment:
We have previously used "at" the same position (lines 10 and 14). "in" actually sounds better to me. Please make consistent.